projects/distributed-tracing/src/pages/trace-detail/trace-detail.service.ts

```diff
@@ -33,7 +42,8 @@ export class TraceDetailService implements OnDestroy {
     this.routeIds$ = route.paramMap.pipe(
       map(paramMap => ({
         traceId: paramMap.get(this.getTraceIdParamName())!,
-        spanId: paramMap.get(this.getSpanIdParamName()) as string | undefined
+        spanId: paramMap.get(this.getSpanIdParamName()) as string | undefined,
+        startTime: paramMap.get(this.getStartTimeParamName()) as string | undefined
```

Comment:
nit: this is returning the value or null and is typed, so rather than cast to get rid of the null, let's actually get rid of it
```suggestion
        startTime: paramMap.get(this.getStartTimeParamName()) ?? undefined
```